index.html

```diff
@@ -1,59 +1,133 @@
+		<link rel="stylesheet" href="index.css">
+		<link rel="icon" type="favicon/x-icon" href="images/favicon.ico">
+		<script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
+   </head>
+
+   <body>
+
+   	<h1>The Coding Space Projects</h1>
+   	<div class="container">
+
+	   	<div class="row tcs-row">
+	   		<div class="col-sm-1"></div>
+		   		<div class="col-sm-10 holder">
+		   			<a id="scratch-img" href="./scratch"><img class="img-responsive" src="./images/scratch2.png" alt="Scratch" /></a>
+		   			<a id="woof-img" href="http://woofjs.com/create"><img class="img-responsive" src="./images/woof2.png" alt="Woof" /></a>
+		   			<a id="web-img" href="./web"><img class="img-responsive" src="./images/web2.png" alt="Scratch" /></a>
+		   		</div>
+		   	<div class="col-sm-1"></div>
+	   	</div>
+
+	   	<div class="row tcs-row hidden-xs">
+	   		<div class="col-sm-1"></div>
+		   		<div class="col-sm-10" id="platform-holder">
+		   			<div class="textbox-holder">
+		   				<div id="scratch-box" class="black-opaque infobox">Program your own interactive stories, games, and animations with Scratch -- and share your creations with others in the online community. <p class="info-para">We recommend starting here if you're new to coding.</p>  </div>
+		   			</div>
+		   			<div class="textbox-holder">
+		   				<div id="woof-box" class="black-opaque infobox">Build complex games, animations, and apps with WoofJS. It uses JavaScript, the language of the web, so you can build for mobile and integrate with APIs and databases. <p class="info-para">We recommend completing two Level 4 Scratch projects before starting with WoofJS.</p></div>
+		   			</div>
+		   			<div class="textbox-holder">
+		   				<div id="web-box" class="black-opaque infobox">Create websites and web apps using HTML, CSS, and JavaScript. <p class="info-para">We recommend completing two Level 3+ WoofJS projects before you start with Web projects.</p></div>
+		   			</div>
+		   		</div>
+		   	<div class="col-sm-1"></div>
+	   	</div>
+	   		
+	   	<div class="row" style="width: 100%">
+	   		<div class="col-sm-1"></div>
+	   		<div class="col-sm-10" id="learn-col">
+	   			<div id="start" class="black-opaque" style="margin-right: 7px">Where do I start?</div>
+	   			<a href="./about.html" target="_blank"><div class="black-opaque" id="about">About this site</div></a>
+	   		</div>
+	   		<div class="col-sm-1"></div>
+	   	</div>
+   		
+	   	<div class="row tcs-row" style="margin: 0">
+	   		<div class="col-sm-1"></div>
+	   		<div class="col-sm-10 holder" style="min-height: 100px">
+	   			<div id="start-box" class="black-opaque" style="font-size: 14px; height: 50%">If you’re new to coding, we recommend starting with a Level 1 Scratch project, like Ghost Busters. If you’re a more experienced coder, we recommend clicking around to find a project that seems challenging for you.</div>
```

Comment:
We recommend starting with a Level 1 Scratch project, like Ghost Busters (link to ghost busters).
(new line)
If you’re a experienced coder, browse the levels to find a project that's challenging for you.